### PR TITLE
add RU locale

### PR DESCRIPTION
**Pull request by daniloff200**
Related to #25 

@eldadfux  could you please add hacktoberfest label to this PR? :)

### CODE REVIEW

**Comment by eldadfux** [comment]
Hey @daniloff200 thank you so much, amazing work and a great contribution to our project!

Would you like me to wait with the merge for 1st of October for it to be eligible as a DigitalOcean PR? (or maybe you have to submit it after 1st of October - not really sure).

Anyway contact me on our Gitter community to get some Appwrite SWAG 🥳.

**Comment by daniloff200** [comment]
@eldadfux  @lex111  done

File: app/config/locale/templates/ru.email.auth.confirm.tpl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        -webkit-font-smoothing: antialiased;
+        font-weight: lighter;
+    }
+</style>
+
+<div style="direction: {{direction}}">
+    Здравствуйте, {{name}},
+    <br />
+    <br />
+    Перейдите по ссылке чтобы подтвердить свой адрес электронной почты.
```
Comment (lex111) [inline app/config/locale/templates/ru.email.auth.confirm.tpl]:
```suggestion
    Перейдите по ссылке, чтобы подтвердить свой адрес электронной почты.
```

File: app/config/locale/templates/ru.email.auth.recovery.tpl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        -webkit-font-smoothing: antialiased;
+        font-weight: lighter;
+    }
+</style>
+
+<div style="direction: {{direction}}">
+    Здравствуйте, {{name}},
+    <br />
+    <br />
+    Перейдите по ссылке чтобы сбросить свой пароль для проекта {{project}} .
```
Comment (lex111) [inline app/config/locale/templates/ru.email.auth.recovery.tpl]:
```suggestion
    Перейдите по ссылке, чтобы сбросить свой пароль для проекта {{project}} .
```

**Comment by daniloff200** [comment]
@lex111  done
cc @eldadfux

**Comment by lex111** [comment]
@daniloff200 please resolve conflict.

**Comment by daniloff200** [comment]
@lex111  +++ done

**Comment by daniloff200** [comment]
fixed some other conflicts
@eldadfux  I think, it's good to merge

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

@lex111 Please reach out to me on Gitter if you would like me to send you some cool Appwrite stickers and magnets!

**Comment by eldadfux** [comment]
@lex111 are you sure your changes were accepted? still see the wrong rtl setting in ru.php

**Comment by lex111** [comment]
>  daniloff200 (Dmitriy Danilov) force-pushed the daniloff200:add-russian-locale  branch from 36dc5ae to 3d93be5 20 hours ago

Not a **single** change of mine :cry:

**Comment by eldadfux** [comment]
@lex111 can you send a new PR? 🤔

**Pull request merged**